apps/rule-manager/app/controllers/RulesController.scala

```diff
@@ -143,8 +143,8 @@ class RulesController(
 
   def canPublish(id: Int) = ApiAuthAction {
     RuleManager.parseDraftRuleForPublication(id, "validate") match {
-      case Right(_)     => Ok
-      case Left(errors) => BadRequest(Json.toJson(errors))
+      case Right(_)     => Ok(Json.toJson(Nil))
```

Comment:
We need to return something here to make it easier to distinguish between these responses on the client-side (now the responses have merged in status code). You could argue for a different approach to what this distinctive marker is - I've opted for an empty list here.